.eslintrc.json

```diff
@@ -24,12 +24,16 @@
     {
       "files": ["*.ts", "*.tsx"],
       "extends": ["plugin:@nx/typescript"],
-      "rules": {}
+      "rules": {
+        "@stylistic/no-extra-semi": "error"
```

Comment:
Was this rule something we had before?

Reply:
My understanding is that this rule was inherited from NX, but with stylistic this is no longer the case and we should thus added it, but I am not 100% sure